<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRow</class>
 <widget class="QDialog" name="EditRow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Row</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QPushButton" name="buttonSave">
     <property name="text">
      <string>Save</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="buttonCancel">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QScrollArea" name="areaColumns">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="areaWidgets">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>464</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
